### PR TITLE
Reverts the crawling nerf.

### PR DESCRIPTION
**Pull request by ShiftyRail**
Reverts #17201.

:cl:

- tweak: There is no longer a delay when crawling.

### CODE REVIEW

File: code/_onclick/other_mobs.dm [hunk 1]
```diff
@@ -53,10 +53,26 @@
...
+
+			"CRAWLING IN MY BRIG!!!!",
+
+			"It's actually about the spirit of a cheeky crawling with the lads",
+
+			"Try to crawl FREELY and AUTOMATICALLY good bye",
+
+			"SNEAKY CR*WLERS!",
+
+			"It's an feature")
```
Comment (jknpj) [inline code/_onclick/other_mobs.dm]: IT'S A FEATUUUURE

**Comment by SonixApache** [comment]
these are all fucking fantastic

**Comment by Diavill** [comment]
this better be the one that doesnt get reverted

**Comment by gbasood** [comment]
i actually thought about merging this but no april fool's day check

File: code/_onclick/other_mobs.dm [hunk 2]
```diff
@@ -53,10 +53,28 @@
...
 
 	if(src.lying && !(isUnconscious() || stunned || paralysis) && check_crawl_ability() && isfloor(A) && isfloor(get_turf(src)) && proximity && !pulledby && !locked_to && !client.move_delayer.blocked())
-		var/delay = round(1 + base_movement_tally()/5) * 1 SECONDS
-		if (do_after(src, A, delay))
-			Move(A, get_dir(src,A), glide_size_override = delay) // So that we're still smooth
-			delayNextMove(delay, additive=1)
+		var/crawldelay = movement_delay()*3
+		Move(A, get_dir(src,A), glide_size_override = crawldelay)
+		delayNextMove(crawldelay, additive=1)
+		if (prob(15))
```
Comment (SonixApache) [inline code/_onclick/other_mobs.dm]: `if(Holiday == APRIL_FOOLS_DAY && prob(15))`

**Pull request merged**